systems/controllers/osc/osc_tracking_data.h

```diff
@@ -69,11 +69,23 @@ class OscTrackingData {
               KinematicsCache<double>& cache_wo_spr,
               const drake::trajectories::Trajectory<double>& traj, double t,
               int finite_state_machine_state);
+  
+  // Getters for debugging
+  Eigen::VectorXd GetY() {return y_;}
+  Eigen::VectorXd GetYDes() {return y_des_;}
+  Eigen::VectorXd GetErrorY() {return error_y_;}
+  Eigen::VectorXd GetDy() {return dy_;}
+  Eigen::VectorXd GetDyDes() {return dy_des_;}
+  Eigen::VectorXd GetErrorDy() {return error_dy_;}
+  Eigen::VectorXd GetDdyDes() {return ddy_des_;}
+  Eigen::VectorXd GetDdyDesConverted() {return ddy_des_converted_;}
+  Eigen::VectorXd GetDdyCommandSol() {return ddy_command_sol_;}
+
   // Getters used by osc block
   Eigen::VectorXd GetOutput() {return y_;}
   Eigen::MatrixXd GetJ() {return J_;}
   Eigen::VectorXd GetJdotTimesV() {return JdotV_;}
-  Eigen::VectorXd GetCommandOutput() {return ddy_command_;}
+  Eigen::VectorXd GetDdyCommand() {return ddy_command_;}
```

Comment:
My notation uses `d` in front of a variable as dot, so dy means the first time derivative of y and ddy means the second time derivative of y. I picked this notation probably because it can shorten the length of the variable names. 

I will add more documentation to make it clear what the variables mean

Reply:
Length should not really be the driving factor in variable naming, readability should be.

Reply:
I've updated the documentation for `OscTrackingData`. I will change the name if you think it's still necessary!